.github/workflows/pr-lint-test-js.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint and tests for JS packages and woocommerce-admin/client
+
+on:
+  pull_request:
+    paths:
+      - 'packages/js/**/**'
+      - 'plugins/woocommerce-admin/client/**'
+      - '!**.md'
+
+jobs:
+  lint-test-js:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.7.0
+        with:
+            access_token: ${{ github.token }}
+      
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      
+      - name: Install PNPM and install dependencies
+        run: npm install -g pnpm && pnpm install
+
+      - name: Lint
+        run: |
+          pnpm nx lint woocommerce-admin
+          pnpm nx lint:js:packages woocommerce-admin
```

Comment:
This command doesn't seem to work for me locally, but I notice it works fine on Github. Do we maybe want to change it to `lint:js-packages`?
I did notice @rjchow comment in Slack ( p1648451119236319-slack-CBKK5KM41 ), but the `run-many` work around didn't work for me either.

Reply:
> This command doesn't seem to work for me locally, but I notice it works fine on Github. Do we maybe want to change it to lint:js-packages?

This is interesting. Both commands can be run in my local. But yes, I think we can change it to `lint:js-packages` to make it work for everyone.

Reply:
Changed in a2327bc90338965ae84689c2dbd7c7a7ed3c7391.